### PR TITLE
[node] Create api-spec.ts

### PR DESCRIPTION
**Pull request by snario**
Some file restructuring to make explicit the public API.

### CODE REVIEW

File: packages/node/src/api-spec.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Node } from "@counterfactual/types";
```
Comment (cf19drofxots) [inline packages/node/src/api-spec.ts]: The name of this file made me think it's a test file at first glance lol

Comment (snario) [reply packages/node/src/api-spec.ts]: Maybe api router is a better name?

Comment (cf19drofxots) [reply packages/node/src/api-spec.ts]: 👍

Comment (cf19drofxots) [reply packages/node/src/api-spec.ts]: I thought the name was gonna be changed before merging? @snario

**Pull request merged**